src/measure_theory/measure_space.lean

```diff
@@ -435,6 +435,9 @@ instance : module ℝ≥0∞ (measure α) :=
 injective.module ℝ≥0∞ ⟨to_outer_measure, zero_to_outer_measure, add_to_outer_measure⟩
   to_outer_measure_injective smul_to_outer_measure
 
+@[simp, norm_cast] theorem coe_nnreal_smul (c : ℝ≥0) (μ : measure α) : ⇑(c • μ) = c • μ :=
```

Comment:
Could you add a specific type ascription here? I don't know whether this is `(c • μ : outer_measure α)` or `(c • μ : α -> ennreal)`

Reply:
Oh, it is the `coe_to_fun` arrow. Still I'd appreciate the explicit typing.

Reply:
The current spelling matches the statement of `coe_smul` a few lines above, and in various places throughout mathlib.

Reply:
Ok, never mind then

bors merge